SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs

```diff
@@ -100,5 +100,37 @@ public List<MpGroupParticipantDto> CreateGroupParticipants(string token, List<Mp
 
             return _ministryPlatformRestRepository.UsingAuthenticationToken(token).Create(mpGroupParticipantDtos, groupParticipantColumns);
         }
+
```

Comment:
Not sure why this functions is needed.  I was thinking we would just set the call number for the second event when we create the participant record.